internal/backend/db/dbgorm/gorm_test.go

```diff
@@ -21,10 +26,72 @@ import (
 	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
-var now time.Time
+var (
+	now    time.Time
+	dbType string
+	gormDB gormdb
+)
+
+func TestMain(m *testing.M) {
```

Comment:
Why do we setup database per suite and per test? This is prone to flaky tests due to left over data.
Is it too slow?

Reply:
I don't know whether u saw last changes. We are setting only once in the beginning. Then we just cleaning DB before each test

Reply:
Cleaning is riskier than a fresh DB, in cleaning you might miss something.